lib/active_graphql/model/relation_proxy.rb

```diff
@@ -169,26 +169,37 @@ def to_graphql
         formatted_action(raw.meta(paginated: true)).to_graphql
       end
 
-      def order(order_params)
-        if order_params.is_a?(Hash)
-          order_by = order_params.keys.first
-          direction = order_params.values.first
-        else
-          order_by = order_params
-          direction = :asc
+      def order(*order_params)
+        order_params_attributes = order_params&.flat_map do |order_param|
```

Comment:
no need for safe navigator since order_params are always array (even if no arguments are given)
```suggestion
        order_params_attributes = order_params.flat_map do |order_param|
```